src/js/dbSNP-import/dbSNP-import-progress.js

```diff
@@ -74,13 +74,29 @@ EvadbSNPImportProgress.prototype = {
                 '<p>This report allows you to track the progress of the dbSNP data import.' +
                 '<p>' +
                     'Variants will be available in the Variant Browser if they satisfy the <a href="?Submit-Data">EVA submission requirements</a>. ' +
-                    'dbSNP variants that don\'t satisfy these requirements will still be imported, and searchable via a separate web view and API. ' +
+                    'dbSNP variants that don\'t satisfy these requirements will still be imported, and searchable via a separate ' +
+                    '<a href="?Home">web view</a> and <a href="https://www.ebi.ac.uk/eva/webservices/identifiers/swagger-ui.html">API</a>, ' +
+                    'and downloadable from our <a href="ftp://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/">FTP</a>. ' +
```

Comment:
I can make explicit the issue about unmapped adding the second sentence:
```suggestion
                    'and downloadable from our <a href="ftp://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/">FTP</a>. ' +
                    'Variants that don't map to any assembly are present only in our FTP at the moment. ' +
```